OmniCore.Model/OmniCore.Model/Interfaces/IMessageExchangeProvider.cs

```diff
@@ -10,6 +10,6 @@ namespace OmniCore.Model.Interfaces
 {
     public interface IMessageExchangeProvider
     {
-        Task<IMessageExchange> GetMessageExchange(IMessageExchangeParameters messageExchangeParameters, IPod pod);
+        Task<IMessageExchange> GetMessageExchange(IMessageExchangeParameters messageExchangeParameters, IPod pod, IMessageExchangeProgress messageProgress);
```

Comment:
design here changed already significantly, is incompatible with current code

Reply:
Base my changes on early 702 code from Git.
Was the easiest way to implement the change for rebuilding the RileyLinkInstance in GetMessageExchange().

(Actually I do not expect you to "merge" anything directly into the current branch.
Mainly trying to report my findings and give you some feedback on possible issues or ways to fix)